SPECS-EXTENDED/libxmlb/libxmlb.spec

```diff
@@ -89,19 +90,85 @@ Executable and data files for installed tests.
 %files tests
 %dir %{_libexecdir}/installed-tests/libxmlb
 %{_libexecdir}/installed-tests/libxmlb/xb-self-test
-%{_libexecdir}/installed-tests/libxmlb/test.xml.gz.gz.gz
+%{_libexecdir}/installed-tests/libxmlb/test.*
 %dir %{_datadir}/installed-tests/libxmlb
 %{_datadir}/installed-tests/libxmlb/libxmlb.test
 
 %changelog
+## START: Generated by rpmautospec
```

Comment:
curious: what is this: `## START`

Reply:
This is Fedora's automation tools. Most of these files have been copied as-is from Fedora's repository. I didn't bother removing these as it's probably easier to maintain with the least amount of changes.